src/index.css

```diff
@@ -17,20 +17,13 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   box-sizing: border-box;
-  background-color: hsl(231, 16%, 92%);
+  /* background-color: hsl(231, 16%, 92%); */
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
 
-#root,
-body,
-html {
-  /* Prevent overscroll on chrome (not supported on safari) */
-  overscroll-behavior-y: none;
-}
-
```

Comment:
I don't think we want to be deleting these lines

Reply:
Wait, do I need to roll back this line manually? This is my first pull request, sorry. :p